src/jsx/import/result/current-result.jsx

```diff
@@ -256,15 +258,15 @@ var CurrentResultDisplay = React.createClass({
                     defaultValue=''
                 />
                 <Submit cssClass='btn' />
-            </form>
+            </form>);
         } else {
-            var resultList = <div className='result-list'>
+            var resultList = (<div className='result-list'>
                 Sorry, no results available!
-            </div>;
+            </div>);
         }
 
-      // display result
-        return(
+        // display result
+        return (
```

Comment:
Seems like you correctly have content on the following, line after the start of parentheses here.